lib/semantics/semantics.cc

```diff
@@ -208,6 +209,70 @@ void SemanticsContext::PopConstruct() {
   constructStack_.pop_back();
 }
 
+void SemanticsContext::SayDoVarRedefine(
+    const Symbol &variable, const parser::CharBlock &location) {
+  const parser::CharBlock doLoc{GetDoVariableLocation(variable)};
+  Say(location, "Cannot redefine DO variable %s"_err_en_US, variable.name())
```

Comment:
Names from the program that are mentioned in error messages are enclosed in single quotes. It makes messages easier to read.

Reply:
Thanks.  I'll add the quotes.